integration-tests/docker/test_env/test_env.go

```diff
@@ -64,10 +69,10 @@ func NewTestEnvFromCfg(cfg *TestEnvConfig) (*CLClusterTestEnv, error) {
 }
 
 func (te *CLClusterTestEnv) ParallelTransactions(enabled bool) {
-	te.Geth.EthClient.ParallelTransactions(enabled)
+	te.EVMClient.ParallelTransactions(enabled)
 }
 
-func (te *CLClusterTestEnv) StartGeth() error {
+func (te *CLClusterTestEnv) StartGeth() (blockchain.EVMNetwork, InternalDockerUrls, error) {
```

Comment:
@iljapavlovs @skudasov why introduce `InternalDockerUrls` since we already have `Geth.InternalHttpUrl` and `Geth.WsUrl`?